packages.md

```diff
@@ -72,66 +135,35 @@ example:
   (:nicknames :ppcre)
 ~~~
 
-## Give a Package a Local Nickname
+### Package Local Nicknames (PLN)
 
-Sometimes it is handy to give a local name to an imported package to save some
-typing, especially when the imported package does not provide nice nicknames.
+Sometimes it is handy to give a local name to an imported package to
+save some typing, especially when the imported package does not
+provide nice nicknames.
 
-This can be achieved by using [`RENAME-PACKAGE`][rename-package]. For example:
+Many implementations (SBCL, CCL, ECL, Clasp, ABCL, ACL, LispWorks >= 7.2…) support Package Local Nicknames (PLN).
 
 ~~~lisp
-(asdf:load-system :cl-ppcre)
-
 (defpackage :mypackage
-  (:use :cl))
-(in-package :mypackage)
-
-;; Add nickname :RE to package :CL-PPCRE.
-(rename-package :cl-ppcre :cl-ppcre '(re))
-
-;; You can use :RE instead of :CL-PPCRE now.
-(re:scan "a" "abc")
-~~~
-
-The function head of `RENAME-PACKAGE` is as follows:
-
-~~~lisp
-(rename-package package-designator name &optional (nicknames nil))
-~~~
-
-As you may guess, this function can be used to rename a package by giving it a
-new *name* instead of a new *nickname*.
-
-However, it is highly recommended **not** to do so. Because otherwise the
-package will be modified and the original name (`:CL-PPCRE` in the example
-above) will **not** be available any more after renaming. For example:
+  (:use :cl)
+  (:local-nicknames (:nickname :original-package-name)
+                    (:alex :alexandria)
+                    (:re :cl-ppcre)))
 
-~~~lisp
-;; DO NOT DO THIS!
-(rename-package :cl-ppcre :re)
+(in-package :mypackage)
 
-;; ERROR!
-(cl-ppcre:scan "a" "abc")
+;; You can use :nickname instead of :original-package-name
+(nickname:some-function "a" "b")
 ~~~
 
-This might cause problems if you want to load another package that relies on
-the modified package.
-
-### Package Local Nicknames (PLN)
-Many implementations (SBCL, CCL, ECL, Clasp, ABCL, ACL, LispWorks >= 7.2) support Package Local Nicknames (PLN). 
+The effect of `PLN` is totally within `mypackage` i.e. the `nickname` won't work in other packages unless defined there too. So, you don't have to worry about unintended package name clash in other libraries.
 
-~~~lisp 
-(defpackage :mypackage 
-  (:use :cl)
-  (:local-nicknames (:nickname :original-package-name)))
-  
-(in-package :mypackage) 
+Prior to package-local-nicknames, users could define *global* ones
+with [`RENAME-PACKAGE`][rename-package]. However, the original name
+would **not** be available any more after renaming to **anyone**. If
+you renamed `cl-ppcre` to `re`, another Quicklisp library that uses
+`cl-ppcre` would fail. Use PLN.
```

Comment:
Saying that the 'original name would not be available' is a little misleading. As `rename-package` can be used to add global nicknames without removing original package name. (As given in original example)
```
;; Add nickname :RE to package :CL-PPCRE.
(rename-package :cl-ppcre :cl-ppcre '(re))
```
I suggest we direct readers to HyperSpec in that matter and warn about not using `rename-package`

```suggestion
Another facility exists for add nicknames to packages. The function [`RENAME-PACKAGE`](http://www.lispworks.com/documentation/HyperSpec/Body/f_rn_pkg.htm) can be used to replace the name and nicknames of a package. But it's use would mean that other libraries may not be able to access the package using the original name or nicknames. There is rarely any situation to use this. Use Package Local Nicknames instead.  
```